example_test.go

```diff
@@ -46,6 +46,12 @@ func ExampleScheduler_StartAsync() {
 	s.StartAsync()
 }
 
+func ExampleScheduler_EveryDuration() {
+	s := gocron.NewScheduler(time.UTC)
+	_, _ = s.EveryDuration(500 * time.Millisecond).Do(task)
```

Comment:
Just a thought:
Can we not make this work under `Every()` itself?

Maybe `Every()` can take `interface{}` as the arg instead of `int`. And internally we can check if it's of type `int` or `time.Duration`

Perhaps we can also allows users to give a string duration like "1s", "5m", "3h" etc

Reply:
That's an interesting idea! Yes, I think we could go this route and would be simpler api with only `Every`.

Reply:
Looks like passing just a number will make it an `int` by default. We might have to typecast all the int types to `uint64`
https://play.golang.org/p/7NM3LiwH03J